frontend/packages/ux-editor/src/types/FormComponent.ts

```diff
@@ -18,10 +18,10 @@ export interface FormComponentBase<T extends ComponentType = ComponentType> {
   handleDeleteElement?: () => void;
   handleUpdateFormData?: (formData: any) => void;
   handleUpdateDataModel?: (dataModelBinding: string) => void;
-  disabled?: boolean;
-  required?: boolean;
-  hidden?: boolean;
-  readOnly?: boolean;
+  disabled?: boolean; // Add expression type?
+  required?: boolean; // Add expression type?
+  hidden?: boolean; // Add expression type?
+  readOnly?: boolean; // Add expression type?
```

Comment:
Yes, we need to do that once we start actually setting expressions on these properties. We should look into how we can import/copy the component types/properties from app-frontend-react so we have correct typings.